envoy/upstream/upstream.h

```diff
@@ -715,6 +715,9 @@ class ClusterTypedMetadataFactory : public Envoy::Config::TypedMetadataFactory {
 
 class TypedLoadBalancerFactory;
 
+using AddressSelectFn = std::function<const Network::Address::InstanceConstSharedPtr(
+    const Network::Address::InstanceConstSharedPtr&)>;
```

Comment:
Please add comments here on what type of address this function takes and what it returns, it's unclear.

Reply:
got it, will do